src/campaigns/ai/util/prompt.util.ts

```diff
@@ -0,0 +1,55 @@
+import { Prisma } from '@prisma/client'
+import { CampaignDetailsContent } from 'src/campaigns/campaigns.types'
+
+type CampaignPositionWithRelations = Prisma.CampaignPositionGetPayload<{
+  include: { topIssue: true; position: true }
+}>
+
+export function positionsToStr(
+  campaignPositions: CampaignPositionWithRelations[],
+  customIssues?: CampaignDetailsContent['customIssues'],
+) {
+  if (!campaignPositions && !customIssues) {
+    return ''
+  }
+  let str = ''
+  campaignPositions.forEach((campaignPosition, i) => {
+    const { position, topIssue } = campaignPosition
+    if (position || topIssue) {
+      str += `Issue #${i + 1}: ${topIssue?.name}. Position on the issue: ${
+        position?.name
+      }. Candidate's position: ${campaignPosition?.description}. `
+    }
+  })
+
+  if (customIssues) {
+    customIssues.forEach((issue) => {
+      str += `${issue?.title} - ${issue?.position}, `
+    })
+  }
+  return str
+}
+
+export function replaceAll(string: string, search: string, replace: string) {
+  const replaceStr = replace || 'unknown'
+  return string.split(`[[${search}]]`).join(replaceStr)
+}
```

Comment:
Is there a reason why we're reinventing `String.prototype.replaceAll` here? 🤔 

I mean.. this could be replaced w/ 

`stringToModify.replaceAll(`[[${search}]]`, replace || 'unknown')` right?